plugins/woocommerce/includes/admin/class-wc-admin-menus.php

```
@@ -290,16 +295,21 @@ public function menu_order( $menu_order ) {
 		// Get index of product menu.
 		$woocommerce_product = array_search( 'edit.php?post_type=product', $menu_order, true );
 
+		// Get index of orders menu.
+		$woocommerce_orders = array_search( 'wc-orders', $menu_order, true );
+
 		// Loop through menu order and do some rearranging.
 		foreach ( $menu_order as $index => $item ) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$index'. (undefined)

(UnusedLocalVariable)